src/App/Home/constants.js

```diff
@@ -0,0 +1,49 @@
+export const entryCards = [
```

Comment:
Maybe use information found in `routes` to do this (specifically `routes/all`) to avoid duplicates?